### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by VSPPedro**
- [x] Add have_one_attached and have_many_attached examples
- [x] Replace is_expected.to with should (I noticed that the project has a preference for should and that is why I am sending this modification suggestion.)

### CODE REVIEW

File: lib/shoulda/matchers/active_record/have_attached_matcher.rb [hunk 1]
```diff
@@ -1,10 +1,52 @@
...
+      #     RSpec.describe User, type: :model do
+      #       it { should have_one_attached(:avatar) }
+      #     end
+      #
+      #     # Minitest (Shoulda)
+      #     class UserTest < ActiveSupport::TestCase
+      #       should have_one_attached(:avatar)
+      #     end
+      #
+      # @return [HaveRichText]
```
Comment (KapilSachdev) [inline lib/shoulda/matchers/active_record/have_attached_matcher.rb:23]:
```suggestion
      # @return [HaveAttachedMatcher]
```

File: lib/shoulda/matchers/active_record/have_attached_matcher.rb [hunk 2]
```diff
@@ -1,10 +1,52 @@
...
+      #     RSpec.describe Message, type: :model do
+      #       it { should have_many_attached(:images) }
+      #     end
+      #
+      #     # Minitest (Shoulda)
+      #     class MessageTest < ActiveSupport::TestCase
+      #       should have_many_attached(:images)
+      #     end
+      #
+      # @return [HaveRichText]
```
Comment (KapilSachdev) [inline lib/shoulda/matchers/active_record/have_attached_matcher.rb:48]:
```suggestion
      # @return [HaveAttachedMatcher]
```

Comment (VSPPedro) [reply lib/shoulda/matchers/active_record/have_attached_matcher.rb:48]: Ops, my bad! Thank you! :sweat_smile:

**Comment by mcmire** [comment]
Thank you for doing this! I completely overlooked the documentation when I merged that PR.

**Pull request merged**